### ISSUE TITLE
PHPStorm on Windows, DDEV in Docker in WSL2, mkcert executable not fount in $PATH

### ISSUE THREAD

**Issue by levonmidoyan** [php-perfect/ddev-intellij-plugin#365]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you sure that this bug is related to this DDEV Integration Plugin?

- [X] I am sure

### Describe the bug

The project doesn't starting via plugin and gives this error:

![image](https://github.com/user-attachments/assets/a7c0bb0d-0dcb-46c0-927c-344f191a0153)

But when I'm running `ddev start` from the terminal the project starting without problems. See below:

 
![image](https://github.com/user-attachments/assets/c83d394a-fb5a-4366-ad98-b97eff5a2f37)

I'm using latest versions:
- ddev: v1.23.4
- PHPStorm: v2024.2.1

Yesterday I've updated all related to wsl. Also I would like to mention that I'm using linuxbrew inside wsl.

**Comment by rfay** [comment]
What does `which -a mkcert` say in your WSL2 shell?

**Comment by rfay** [comment]
Can you please show the full `ddev start` that is shown in the PhpStorm window? Some of it is obscured. Is it showing the ddev path as /home/linuxbrew/.linuxbrew/bin/ddev ?

I'm not sure why you'd get an error anyway, as DDEV can work without mkcert.

**Comment by levonmidoyan** [comment]
Yes the path is `/home/linuxbrew/.linuxbrew/bin/ddev` but I didn't configure any setting the plugin did that itself.

This one from form the terminal

![image](https://github.com/user-attachments/assets/8c8b068b-817b-4ac7-86e6-09b7b97b8e37)

And this one run DDEV plugin

![image](https://github.com/user-attachments/assets/6bef91ca-37c0-4dea-b850-7f85284f0dad)

**Comment by rfay** [comment]
This is the code in question, https://github.com/ddev/ddev/blob/ddddc6ab5ade15563bc8cae943f9e051f11a0a1c/pkg/ddevapp/traefik.go#L335-L338

```go
		out, err := exec.RunHostCommand("mkcert", c...)
		if err != nil {
			util.Failed("Failed to create certificates for project, check mkcert operation: %v; err=%v", out, err)
		}
```

OH, I think I see. You don't have /home/linuxbrew/.linuxbrew/bin in your PATH. Instead, you may have manually configured the `ddev` binary to be /home/linuxbrew/.linuxbrew/bin/ddev

The fix is to make sure that `/home/linuxbrew/.linuxbrew/bin` is in your $PATH. This is a part of the normal setup of linuxbrew. 

However, I don't really recommend using Linuxbrew for DDEV installation. You're better off using the techniques in the docs, which use apt
* https://ddev.readthedocs.io/en/stable/#system-requirements-windows-wsl2
* https://ddev.readthedocs.io/en/stable/#system-requirements-linux

**Comment by levonmidoyan** [comment]
@rfay thank you for the explanations. I'll try to do so.

**Comment by levonmidoyan** [comment]
There is no need to uninstall `ddev`, I've checked the $PATH and all was fine `/home/linuxbrew/.linuxbrew/bin` was there.
And after I just reinstalled `mkcert` and plugin started project successfully.
As `mkcert` was installed via `brew` I reinstalled it using apt technique and now `mkcert` in `/usr/bin`.

@rfay thank you for your support. I would like to suggesting you to improve the plugin to handle `linuxbrew` problems like this and if there will be a solution that would be great.

As my issue is solved I'm closing the issue.

**Comment by rfay** [comment]
Glad it's working! Lots of people use ddev with linuxbrew still, I don't think this problem had to do with the extension here, it had to do with the PATH that PhpStorm inherited from you somehow.
